.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
I don't think this belongs here.

Reply:
this is local file, which MacOS creates to store custom attributes of its containing folder (position of icons, etc.).  when I create local GIT copy, this file is always coming up and I assume it happens to other developers (who use Mac). 
but yes, having perused all content of .gitignore I can see that it contains wildcards to files related to Curl itself.  I will remove this now and check is there way to configure GIT locally to ignore some files.

Reply:
ok, I found solution - I can have my own gitignore using:

git config --global core.excludesfile ~/.gitignore

Reply:
Maybe explicitly ignoring all dot-files + whitelisting those we need wouldn't be a bad idea?

Reply:
I don't see a need for that. I think we should stick to ignoring files that are generated by a curl build. If you want to ignore more files, it is very easy to do what @ajorajev did and just set your own ignore files for those - I do that too.

Reply:
I thought the same before I've spotted Vim, Eclipse, and backup files typical for Pluma/GEdit/etc there. But I won't argue on it.